core/templates/dev/head/pages/exploration-player-page/learner-experience/learner-answer-info-card.directive.html

```diff
@@ -0,0 +1,43 @@
+<div class="learner-answer-info-card">
+  <p class="question-content">Hey, how did you landed on this answer?</p>
```

Comment:
how did you land.
Also, @DubeySandeep I think all these strings should be a part of I18N translations, right?

Reply:
I agree this is a learner facing content, make it go through i18n. (You can check #5997 for help.)

Reply:
Just working on it.

Reply:
This is still pending.

Reply:
Done